### PR TITLE
docs(options): mention how to disable `inccommand` preview

### PR DESCRIPTION
**Pull request by gtlsgamr**
Added blank option for inccommand in case the user wants to disable the incremental command preview.

### CODE REVIEW

File: runtime/doc/options.txt [hunk 1]
```diff
@@ -3228,6 +3228,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	"nosplit": Shows the effects of a command incrementally, as you type.
 	"split"	 : Also shows partial off-screen results in a preview window.
+	""	 : Does not preview the changes directly.
```
Comment (clason) [inline runtime/doc/options.txt:3231]:
```suggestion
	""       : Disable preview.
```

**Comment by gpanders** [comment]
I think this is a good opportunity to redo the docs for `'inccommand'` completely. Compare it to e.g. `'switchbuf'` (another option that takes an (optional) string). The latter is more in line with traditional Vim option documentation, and I would support rewriting the inccommand docs to follow that style.

**Comment by gpanders** [comment]
I think this is a good opportunity to redo the docs for `'inccommand'` completely. Compare it to e.g. `'switchbuf'` (another option that takes an (optional) string). The latter is more in line with traditional Vim option documentation, and I would support rewriting the inccommand docs to follow that style.

My suggestion:

```
						*'inccommand'* *'icm'*
'inccommand' 'icm'	string	(default "nosplit")
			global

	When nonempty, shows the effects of |:substitute:|, |:smagic|, and
	|:snomagic| as you type.

	Possible values:
	   nosplit	Shows the effects of a command incrementally, as you
			type.
	   split	Also shows partial off-screen results in a preview
			window.

	If the preview is too slow (exceeds 'redrawtime') then 'inccommand' is
	automatically disabled until |Command-line-mode| is done.
```

**Comment by clason** [comment]
@gpanders I've pushed a commit with your suggested change.

**Review by gpanders — APPROVED** [review]

**Pull request merged**